README.md

```diff
@@ -0,0 +1,7 @@
+[![Build Status](https://travis-ci.org/andela/ah-fulldeck-frontend.svg?branch=ch-update-travisCI-162948949)](https://travis-ci.org/andela/ah-fulldeck-frontend)
```

Comment:
@JoshuaKodhe Change the branch to develop

Reply:
> @JoshuaKodhe Change the branch to develop

@victorjambo I have changed the badge but the build is still unknown, changes for the badge will occur once the branch is merged into develop branch.